alexey.fesenko/task_2.rb

```diff
@@ -0,0 +1,3 @@
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+arr = hash.select { |k| k.to_s.downcase.start_with? "yes" }.keys
+p arr.length
```

Comment:
nitpick: Also there is a way to refactor to use https://apidock.com/ruby/Enumerable/count so your code will be a little more compact

Reply:
As there are no major issues here, I'll give you some time to apply fixes, if you want. This request will be merged in 2 days

Reply:
@Flippylolz  Sorry, but would you mind providing more wide answer to the  "Also there is a way to refactor to use https://apidock.com/ruby/Enumerable/count so your code will be a little more compact", cause, unfortunately, this point is not very clear for me at least for the right now. Thanks in advance.

Reply:
Hey @Santo555, you can pass a block to count method, so it will count only things you need. For example 
```
ary = [1, 2, 4, 2]
ary.count{ |x| x.even? } #=> 3
ary.count(&:even?) #=> 3
```